### PR TITLE
add document of `SHOW PLUGINS`

### PR DESCRIPTION
**Pull request by imtbkcat**
### What is changed, added or deleted? (Required)
Add document of `SHOW PLUGINS`

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: sql-statements/sql-statement-show-plugins.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+title: SHOW PLUGINES
+summary: TiDB 数据库中 SHOW PLUGINS 的使用概况。
+category: reference
+aliases: ['/docs-cn/dev/reference/sql/statements/show-pluhins/']
```
Comment (jackysp) [inline sql-statements/sql-statement-show-plugins.md:5]:
```suggestion
aliases: ['/docs-cn/dev/reference/sql/statements/show-plugins/']
```

Comment (jackysp) [reply sql-statements/sql-statement-show-plugins.md]: it should be removed

File: sql-statements/sql-statement-show-plugins.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+---
+title: SHOW PLUGINES
```
Comment (yikeke) [inline sql-statements/sql-statement-show-plugins.md:2]:
```suggestion
title: SHOW PLUGINS
```

**Comment by bobotu** [comment]
PTAL @yikeke

File: sql-statements/sql-statement-show-plugins.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+---
+title: SHOW PLUGINS
+summary: TiDB 数据库中 SHOW PLUGINS 的使用概况。
+category: reference
+---
+
+# SHOW PLUGNS
```
Comment (yikeke) [inline sql-statements/sql-statement-show-plugins.md:7]:
```suggestion
# SHOW PLUGINS
```

**Pull request merged**